### DISCUSSION TITLE
Persist sort order

### DISCUSSION THREAD

**Discussion by manuelteuber** [Power-Components/livewire-powergrid#1328] [Ideas]
# ⚡ PowerGrid ⚡ Feature Request

## Summary

It would be great if the sort order could be persisted, like filters and columns.

### Why is this needed?

For better UX.

### Suggested Solution(s)

Extend the persisted state to include sort order.

### Related issue(s)/PR(s)

-

**Comment by luanfreitasdev** [comment]
https://github.com/Power-Components/livewire-powergrid/pull/1387

**Reply by luanfreitasdev** [reply]
Hello @manuelteuber . This is being implemented. Thank you!
